sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/Status.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.CosmosDB.Models
+{
+    /// <summary> Enum to indicate current buildout status of the region. </summary>
+    public readonly partial struct Status : IEquatable<Status>
```

Comment:
`Status` is too generic, consider renaming to `CosmosDBBuildoutStatus` or some other name makes sense.

Reply:
added CosmosDBStatus